packages/client/tests/functional/_utils/setupTestSuiteEnv.ts

```diff
@@ -116,18 +115,19 @@ export async function setupTestSuiteDatabase(
   try {
     const consoleInfoMock = jest.spyOn(console, 'info').mockImplementation()
     const dbPushParams = ['--schema', schemaPath, '--skip-generate']
-    const providerFlavor = suiteConfig.matrixOptions['providerFlavor'] as ProviderFlavor | undefined
-    // `--force-reset` is great but only using it where it's necessary makes the
-    // tests faster Since we have full isolation of tests / database, we do not
-    // need to force reset but we currently break isolation for Vitess (for
-    // faster tests), so it's good to force reset in this case
-    if (providerFlavor === ProviderFlavors.VITESS_8) {
+
+    // we reuse and clean the db when running in single-threaded mode
+    if (process.env.JEST_MAX_WORKERS === '1') {
```

Comment:
Do `process.env.JEST_MAX_WORKERS` and `--runInBand` achieve the same goals? If so, could we just use one of those to avoid redundancy?

Reply:
JEST_MAX_WORKERS provides the information for us to know if it's run in band.